cocos/scripting/lua-bindings/manual/cocos2d/lua_cocos2dx_manual.cpp

```diff
@@ -2403,7 +2403,7 @@ int tolua_cocos2d_Node_setAnchorPoint(lua_State* tolua_S)
         if (!ok)
             return 0;
 
-        cobj->setAnchorPoint(cocos2d::Vec2(x,y));
+        cobj->setAnchorPoint(cocos2d::Vec2((float)x,(float)y));
```

Comment:
This convert is not needed. The reason is the same as above.

Reply:
`x` and `y` are double, need convert to `float`.